packages/js/examples/react-audio/stories/components/Utilities.js

```diff
@@ -108,6 +108,65 @@ function Utilities({ environment, username, password }) {
     });
   };
 
+  const setAudioSettings = async () => {
+    const audioInList = await clientRef.current.getAudioInDevices();
+    const deviceId = audioInList[0] ? audioInList[0].deviceId : '';
+    const label = audioInList[0] ? audioInList[0].label : '';
+
+    const settings = {
+      micId: deviceId,
+      micLabel: label,
+      echoCancellation: true,
+    };
+
+    const results = await clientRef.current
+      .setAudioSettings(settings)
+      .catch((error) => console.log(error));
+
+    setLog({
+      title: (
+        <span>
+          Returns the audio settings applied for <b>{label}</b>
+        </span>
+      ),
+      message: (
+        <pre style={{ display: 'block', backgroundColor: '#ccc' }}>
+          {JSON.stringify(results, undefined, 2)}
+        </pre>
+      ),
+    });
+  };
```

Comment:
This is the duplicated code from https://github.com/team-telnyx/webrtc/pull/73/files